multiversx_sdk_cli/cli_transactions.py

```diff
@@ -85,3 +95,27 @@ def get_transaction(args: Any):
     transaction = proxy.get_transaction(args.hash)
     output = CLIOutputBuilder().set_transaction_on_network(transaction, omit_fields).build()
     utils.dump_out_json(output)
+
+
+def sign_transaction(args: Any):
+    args = utils.as_object(args)
+
+    cli_shared.check_guardian_args(args)
+    cli_shared.check_broadcast_args(args)
+
+    tx = Transaction.load_from_file(args.infile)
+    if args.guardian:
+        cli_shared.check_options_for_guarded_tx(tx.options)
+
+    # empty signatures for user signing
+    tx.guardianSignature = ""
+    tx.signature = ""
+
+    account = cli_shared.prepare_account(args)
+    signature = account.sign_transaction(tx)
+    tx.signature = signature
```

Comment:
Can be written as `tx.signature = account.sign_transaction(tx)`.

Reply:
Now written as above.